scripts/gm/bin/gm.toml

```diff
@@ -0,0 +1,6 @@
+[global]
```

Comment:
Probably should not be merged with this hardcoded values. Please rm this file from the PR.

Reply:
Done!